develop/reference/articles/01-intro/00-development-reference-intro.markdown

```diff
@@ -116,16 +64,31 @@ models and create powerful UI components in your presentation layer.
 
 <p style="margin-left: 40px;">
 <span style="font-size:18px;">
-<a href="http://docs.liferay.com/portal/7.0/taglibs/" target="_blank">
-Portal Taglibs <span class="opens-new-window-accessible">(Opens New Window)</span>
+<a href="https://docs.liferay.com/portal/7.0/taglibs/" target="_blank">
+Portal Taglibs&nbsp;<span class="opens-new-window-accessible">(Opens New Window)</span>
+</a>
+</span>
+</p>
+
+<p style="margin-left: 40px;">
+Describes Liferay Portal 7.0 CE tag libraries at the time of the latest GA release. You can
+also download them from the 
+<a href="https://www.liferay.com/downloads" target="_blank">Downloads page<span
+class="opens-new-window-accessible"> (Opens New Window)</span></a>.
+</p>
+
+<p style="margin-left: 40px;">&nbsp;</p>
+
+<p style="margin-left: 40px;">
+<span style="font-size:18px;">
+<a href="https://docs.liferay.com/portal/7.0/javadocs/" target="_blank">
+Module Taglibs for All Versions&nbsp;<span class="opens-new-window-accessible">(Opens New Window)</span>
 </a>
 </span>
 </p>
 
 <p style="margin-left: 40px;">
-Tag libraries for <a href="http://alloyui.com/" target="_blank">AlloyUI <span
-class="opens-new-window-accessible">(Opens New Window)</span></a>, Liferay UI,
-Liferay themes, Liferay portlets, Liferay security, and standard portlets.
+Taglib doc JARs for <em>all</em> versions of CE modules are available here.
 </p>
 
 <p style="margin-left: 40px;">&nbsp;</p>
```

Comment:
GitHub doesn't allow me to add comments for lines below this one. This comment is about: liferay-plugin-package_7_0_0.properties

I don't think that file is used for anything (or at least not anything relevant), so I think it's better to not include it here.